static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -516,22 +532,52 @@ export function getMergedTasks({organization, projects, onboardingContext}: Opti
   }));
 }
 
-const PulsingIndicator = styled('div')`
+const PulsingIndicator = styled('div')<{hasQuickStartUpdatesFeature?: boolean}>`
   ${pulsingIndicatorStyles};
-  margin-right: ${space(1)};
+  ${p =>
+    p.hasQuickStartUpdatesFeature
+      ? css`
+          margin: 0 ${space(0.5)};
+        `
+      : css`
+          margin-right: ${space(1)};
+        `}
 `;
 
 const EventWaitingIndicator = styled(
-  (p: React.HTMLAttributes<HTMLDivElement> & {text?: string}) => (
-    <div {...p}>
-      <PulsingIndicator />
-      {p.text || t('Waiting for event')}
-    </div>
-  )
+  (
+    p: React.HTMLAttributes<HTMLDivElement> & {
+      hasQuickStartUpdatesFeature: boolean;
+      text?: string;
+    }
+  ) => {
+    if (p.hasQuickStartUpdatesFeature) {
+      return (
+        <div {...p}>
```

Comment:
You are spreading `text` & `hasQuickStartUpdatesFeature` onto the div . We should omit them.